docs/developer-docs/latest/setup-deployment-guides/configurations/required/server.md

```diff
@@ -32,6 +32,7 @@ The `./config/server.js` file can include the following parameters:
 | `cron.tasks`                        | Declare [CRON jobs](/developer-docs/latest/setup-deployment-guides/configurations/optional/cronjobs.md) to be run at specific dates.                                                                                                                                                                                                                                        | object            |                     |
 | `dirs`                              | Path configuration of different directories Strapi uses.                                                                                                                                                                                                                                                                                                                    | object            |                     |
 | `dirs.public`                       | Customize the path of the public folder.                                                                                                                                                                                                                                                                                                                                    | string            | `./public`          |
+| `webhooks.populateRelations`        | For compatibility reasons, it is set to `true`. We recommend setting it to `false` if you don't need it.                                                                                                                                                                                                                           | boolean           | `true`              |
```

Comment:
```suggestion
| `webhooks.populateRelations`        | For backward compatibility reasons, the default value is `true`, but the recommended value is `false` to avoid performance issues.                                                                                                                                                                                                                           | boolean           | `true`              |
```